### PR TITLE
"FS-1094 refactoring"

### PR DESCRIPTION
**Pull request by cyeniceri**
Reverts hmrc/single-customer-account-stub#18

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -19,13 +17,8 @@ lazy val microservice = Project(appName, file("."))
       compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
       "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
     ),
-    PlayKeys.playDefaultPort := 8421,
-    ScoverageKeys.coverageMinimumStmtTotal := 30,
```
Comment (yangwong9664) [inline build.sbt:23]: why is the coverage being removed? same comment as the wrapper-data PR

File: conf/prod.routes [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Add all the application routes to the app.routes file
-->         /single-customer-account-stub  app.Routes
```
Comment (yangwong9664) [inline conf/prod.routes:2]: removing app.routes might be problematic, if another service has the same endpoint in one of the environments it would clash

**Review by yangwong9664 — CHANGES_REQUESTED** [review]

File: conf/prod.routes (hunk 2 above)
Comment (cyeniceri) [reply conf/prod.routes:2]: the point here is, we should have same endpoint except host+port. By doing this, we can override the environment variables for each external service(api), we can achieve real(qa,prod) or stub data (local, dev, staging).

File: build.sbt (hunk 1 above)
Comment (cyeniceri) [reply build.sbt:23]: CodeCoverageSettings file provide all the required parameters.

**Review by kritikarai6 — APPROVED** [review]

**Pull request merged**